minkindr_gtsam/include/kindr/minimal/cubic-hermite-quaternion-gtsam.h

```diff
@@ -0,0 +1,92 @@
+#ifndef MINKINDR_QUATERNION_CUBIC_HERMITE_GTSAM_H
+#define MINKINDR_QUATERNION_CUBIC_HERMITE_GTSAM_H
```

Comment:
The functions in this file are great, but how can we call them to evaluate the curve outside of GTSAM?